packages/turbo/ios/RNSessionConfiguration.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  RNSessionConfiguration.swift
+//  RNTurbo
+//
+//  Created by Patryk Klatka on 22/04/2024.
+//
+
+import Foundation
+
+class RNSessionConfiguration {
+  var sessionDelegate: RNSession? = nil {
+    didSet {
+      updateConfiguration()
+    }
+  }
+
+  var webViewConfiguration: WKWebViewConfiguration = WKWebViewConfiguration()
+  var isScrollEnabled: Bool = true {
+    didSet {
+      updateConfiguration()
+    }
+  }
+    
+  func updateConfiguration() {
+    sessionDelegate?.webView.scrollView.isScrollEnabled = isScrollEnabled
```

Comment:
I'm wondering if the access to the scroll view via session is needed at all?

Couldn't we just do:

```
controller!.visitableView.webView?.isScrollEnabled = ...
```